newIDE/app/src/ProjectManager/index.js

```diff
@@ -882,6 +939,8 @@ export default class ProjectManager extends React.Component<Props, State> {
                         eventsFunctionsExtension
                       )
                     }
+                    addLabel={'Add a New Extension'}
+                    onAdd={() => this._addEventsFunctionsExtension(name, i)}
```

Comment:
Just thinking about it now, `name` arguments are actually not used at all right? Can you remove them? :)

Reply:
yes